modules/client-configuration/pages/snippets/add_channels_scc_multiarch.adoc

```diff
@@ -0,0 +1,18 @@
+.Procedure: Adding Software Channels at the Command Prompt
+. At the command prompt on the {productname} Server, as root, use the [command]``spacewalk-common-channels`` command to add the appropriate channels.
+Ensure you specify the correct architecture:
++
+----
+spacewalk-common-channels \
```

Comment:
If this SCC, why spacewalk-common-channels?

Reply:
see above